kubetest/azure.go

```diff
@@ -400,8 +400,12 @@ func (c *Cluster) buildHyperKube() error {
 	}
 	log.Println("Docker login success.")
 	log.Println("Building hyperkube.")
+
 	pushHyperkube := util.K8s("kubernetes", "hack", "dev-push-hyperkube.sh")
-	if err1 := control.FinishRunning(exec.Command(pushHyperkube)); err1 != nil {
+	cmd = exec.Command(pushHyperkube)
+	// dev-push-hyperkube will produce a lot of output to stdout. We should capture the output here.
```

Comment:
is it using any of the standard kubernetes repo build tooling underneath? like the make or the bazel builds? those have some flags / env to tune the verbosity.

EG
https://github.com/kubernetes-sigs/kind/blob/master/pkg/build/kube/dockerbuildbits.go#L101-L102